content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -160,9 +160,9 @@ You can use `NotIn` and `DoesNotExist` to achieve node anti-affinity behavior, o
 If you specify both `nodeSelector` and `nodeAffinity`, *both* must be satisfied for the pod
 to be scheduled onto a candidate node.
 
-If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types, then the pod can be scheduled onto a node **only if all** `nodeSelectorTerms` can be satisfied.
+If you specify multiple `nodeSelectorTerms` associated with `nodeAffinity` types, then the pod can be scheduled onto a node **if one of the** `nodeSelectorTerms` can be satisfied.
```

Comment:
You're right. The API clearly says it's ORed:

https://github.com/kubernetes/kubernetes/blob/77bf3135842e608f6771c05e25a283068f974bb3/staging/src/k8s.io/api/core/v1/types.go#L2536

A little surprised about the issue..

Reply:
Ah, the regression is here: #17557.